CHANGELOG.md

```diff
@@ -24,8 +24,9 @@ No changes to highlight.
 * Fixed some embedded demos in the guides by not loading the gradio web component in some guides by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 2403](https://github.com/gradio-app/gradio/pull/2403) 
 * When an `Image` component is set to `source="upload"`, it is now possible to drag and drop and image to replace a previously uploaded image by [@pngwn](https://github.com/pngwn) in [PR 2400](https://github.com/gradio-app/gradio/pull/2410)
 * Improve documentation of the `Blocks.load()` event by [@abidlabs](https://github.com/abidlabs) in [PR 2413](https://github.com/gradio-app/gradio/pull/2413)
-* Decreased latency in iterative-output demos by making the iteration asynchronous [@freddyaboulton](https://github.com/freddyaboulton) in [PR 2409](https://github.com/gradio-app/gradio/pull/2409)
+* Decreased latency in iterative-output demos by making the iteration asynchronous by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 2409](https://github.com/gradio-app/gradio/pull/2409)
 * Automatically restart spaces if they're down by [@aliabd](https://github.com/aliabd) in [PR 2405](https://github.com/gradio-app/gradio/pull/2405)
+* Build Gradio from source in ui tests by by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 2440](https://github.com/gradio-app/gradio/pull/2440) 
```

Comment:
Open question but is the changelog for the repo, or just for the library?

Reply:
I interpret it as the repo. I can remove this line though since it does not impact users of the library.

Reply:
I don't mind but would be good to decide if we want it to be for the lib or for the repo. If we decide both then we should have two changelogs because the changelog is the changelog for the library first and foremost and should only include changes that impact library users.

Reply:
Makes sense - noted in #2431